src/PrestaShopBundle/Form/ErrorMessage/Factory/AdministrationConfigurationErrorFactory.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\ErrorMessage\Factory;
+
+use PrestaShop\PrestaShop\Core\Form\ErrorMessage\AdministrationConfigurationError;
+use PrestaShop\PrestaShop\Core\Form\ErrorMessage\ConfigurationErrorInterface;
+use PrestaShop\PrestaShop\Core\Form\ErrorMessage\Factory\ConfigurationErrorFactoryInterface;
+use PrestaShopBundle\Entity\Repository\LangRepository;
+use PrestaShopBundle\Form\Admin\Configure\AdvancedParameters\Administration\GeneralDataProvider;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class AdministrationConfigurationErrorFactory implements ConfigurationErrorFactoryInterface
+{
+    /**
+     * @var TranslatorInterface
+     */
+    protected $translator;
+
+    /**
+     * @var LangRepository
+     */
+    protected $langRepository;
+
+    public function __construct(
+        TranslatorInterface $translator,
+        LangRepository $langRepository
+    ) {
+        $this->translator = $translator;
+        $this->langRepository = $langRepository;
+    }
+
+    /**
+     * @param ConfigurationErrorInterface $error
+     * @param string $label
+     *
+     * @return string
+     */
+    public function getErrorMessageForConfigurationError(ConfigurationErrorInterface $error, string $label): ?string
```

Comment:
Could you add unit test for checking the behavior ? Thanks

Reply:
Added the test